components/inquiry_questionnaire/map/test_steps.coffee

```diff
@@ -0,0 +1,71 @@
+module.exports = [
+  { is_logged_out_but_has_account: true: ['account'] }
+  pre_qualify: {
+    true: [
+      { has_seen_commercial_interest: false: ['commercial_interest'] }
+      {
+        is_collector:
+          true: [
+            { has_basic_info: false: ['basic_info'] }
+            { has_seen_artists_in_collection: false: ['artists_in_collection'] }
+            { has_seen_galleries_you_work_with: false: ['galleries_you_work_with'] }
+            { has_seen_auction_houses_you_work_with: false: ['auction_houses_you_work_with'] }
+            { has_seen_fairs_you_attend: false: ['fairs_you_attend'] }
+            { has_seen_institutional_affiliations: false: ['institutional_affiliations'] }
+            'test_inquiry'
```

Comment:
The two different views here are `test_inquiry` and `test_account`. `test_inquiry` is identical but instead of saving the inquiry before moving on, it stores it in `State`. `test_account` picks this back up, and if the user successfully logs in (either by registration or by normal login) it will go ahead and save the inquiry.